classes/LwcSalagaController.cls

```diff
@@ -0,0 +1,136 @@
+public with sharing class LwcSalagaController {
+	public static final String FIRST_NAME_API_NAME = 'FirstName';

```

Comment:
What do you think, what it diff between this static string and String.valuesOf(User.FirstName)?